src/quill-cursors/quill-cursors.ts

```diff
@@ -119,14 +119,15 @@ export default class QuillCursors {
 
     cursor.show();
 
+    const containerRectangle = this._quill.container.getBoundingClientRect();
```

Comment:
Can we please copy the Quill Bubble behaviour and use [`options.bounds`][1], falling back to [`document.body`][2]. This would keep the behaviour consistent with Quill tooltips.

[1]: https://github.com/quilljs/quill/blob/v1.3.7/themes/bubble.js#L23
[2]: https://github.com/quilljs/quill/blob/v1.3.7/ui/tooltip.js#L4

Reply:
Why to document.body? As we have overflow hidden on container, we should use it, not a body. Am I missing something?

Reply:
Yes, that's a fair point. I was trying to see if I could remove `overflow: hidden`, but I do think we need it for the fixed-height editor case.

I'd wanted behaviour to stay as close to Quill Bubble as possible, but I think Quill's behaviour is actually broken in the fixed-height case anyway (see below), so yes, let's fall back to `.ql-container` if no `bounds` are defined.

<img width="1207" alt="Screen Shot 2020-01-24 at 12 17 18" src="https://user-images.githubusercontent.com/12036746/73068606-87f82780-3ea3-11ea-922e-07297616e9fc.png">

Reply:
Just gave it another thought - which cases do you see, when custom bounds could be necessary? 
I don't mind having it in option, but as anyway - there is no sense in setting it to something bigger than container as container will clip it anyway and on the other hand inside container it is the Quill world, so I don't think one can use somethign from there.

Reply:
I'm not really sure, but given that Quill offers it as an option, I think we should honour it, too. For example, I guess the following is theoretically possible:

  - use a full-height editor (no scroll)
  - override the `overflow: hidden` (because we don't need it with no scroll)
  - have your editor wrapped in a bigger container
  - set the container as the bounds, so that the flag can fit to the right-hand side in more situations

It's a bit contrived. I mostly just think we should stay consistent with Quill.